src/sidebar/components/MarkdownEditor.tsx

```diff
@@ -80,6 +82,7 @@ const SHORTCUT_KEYS: Record<Command, string> = {
   math: 'm',
   numlist: 'o',
   quote: 'q',
+  mention: 'a', // TODO
```

Comment:
We don't have to have a shortcut for every toolbar icon. `Ctrl+A` is select-all, which I would avoid overriding. It might be necessary to do some refactoring to allow some icons not to have shortcuts.

Reply:
> It might be necessary to do some refactoring to allow some icons not to have shortcuts.

I just had to adjust a type, but the component already defines the shortcut as an optional prop and only adds it if provided.